src/librustc/session/config.rs

```diff
@@ -1027,6 +1027,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "add a source pattern to the file path remapping config"),
     remap_path_prefix_to: Vec<String> = (vec![], parse_string_push, [TRACKED],
         "add a mapping target to the file path remapping config"),
+    force_unstable_if_unmarked: bool = (false, parse_bool, [TRACKED],
+        "force all crates to be `rustc_private` unstable"),
```

Comment:
We could, yeah, but I'm not sure it'd buy much? We otherwise would still hardcode the issue number as well as the description of the unstable feature (the "note"), and passing all that info as CLI flags may not also be worth it :(.

Do you think we'll want different unstable features on differnet crates though?